src/app/functions/server/getExperiments.ts

```diff
@@ -22,8 +22,14 @@ export async function getExperiments(
   let features;
   try {
     features = await fetch(`${serverUrl}/v1/features/`, {
+      headers: {
+        "Content-Type": "application/json",
+      },
       method: "POST",
-      body: JSON.stringify({ client_id: userId, context: {} }),
+      body: JSON.stringify({
+        client_id: userId,
+        context: { key: "example-key" },
```

Comment:
Q: should this be an env var? Or is this not really _"used"_ (as long as it is non-empty)?

Reply:
Right it's just an example, but it will be populated by session-specific info that Nimbus can use for targeting (language, geo-location, etc etc)